From 5de065eabfba338acd9bd600072ac115fa4683db Mon Sep 17 00:00:00 2001
From: Ihor Usyk <ihor_usyk@epam.com>
Date: Wed, 23 Jun 2021 13:00:04 +0000
Subject: [PATCH] connman main.conf: add 'can,lo,vif,xenbr,eth' in black list

Otherwise, there is  a conflict with networkd and internet does not work

Change-Id: I84cb04b0f3a609e16f657f5108acbe84f472ba8b
Signed-off-by: Ihor Usyk <ihor_usyk@epam.com>
---
 meta-agl-profile-core/recipes-connectivity/connman/files/main.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meta-agl-profile-core/recipes-connectivity/connman/files/main.conf b/meta-agl-profile-core/recipes-connectivity/connman/files/main.conf
index 4dba25b..b18f390 100644
--- a/meta-agl-profile-core/recipes-connectivity/connman/files/main.conf
+++ b/meta-agl-profile-core/recipes-connectivity/connman/files/main.conf
@@ -1,3 +1,3 @@
 [General]
 FallbackTimeservers=pool.ntp.org
-NetworkInterfaceBlacklist=vmnet,vboxnet,virbr,ifb,meth
+NetworkInterfaceBlacklist=can,lo,vif,xenbr,eth,vmnet,vboxnet,virbr,ifb,meth
-- 
2.7.4

